data/transit/route/bus.json

```
@@ -16197,6 +16197,15 @@
         "route": "bus"
       }
     },
+    {
+      "displayName": "Transjakarta",
+      "locationSet": {"include": ["id"]},
```

```suggestion
      "locationSet": {"include": ["id"]},
      "matchNames":["TJ"],
```

> Also add key "TJ" for search that leads to those features. Thanks

You said this in #9147, which can be achieved by add this line or add `network:short:TJ` in tags